readme.txt

```diff
@@ -28,6 +28,9 @@ A very useful plugin to display your mautic contacts count on your website. This
 
 == Changelog ==
 
+=  Version 1.0.5  = 
```

Comment:
@nileshc-bsf If there are no changes in the code, then there is no need to add a changelog and no need to update the version number as well.

We can just update the: 

`Tested up to: 5.6`

Please refer to this PR - https://github.com/brainstormforce/reset-astra-customizer/pull/19/files

Also, can you please add the `.github/workflows/phpcs.yml` file to check coding standards using GitHub action in this plugin. 

Please let me know if you have any doubts.

Reply:
@premanshup - Changes Done and verified